.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -1,21 +1,31 @@
 ---
-name: Feature Request
-about: For Requesting enhancement or changes to API functionality
-
+name: 💡 Feature Request
+about: For requesting new features or enhancements on the VA API Platform
 ---
 
-## APIs Affected
+Important note: As this is a public forum, please do not include your VA API OAuth Client ID, VA API OAuth Client Secret, VA API key, or any Personally Identifiable Information (PII) or Protected Health Information (PHI) in your request.
+
+
+## Email address
+
+<!-- Please include your email address. -->
+
+
+## APIs affected
+
+<!-- Please list affected APIs (if applicable). -->
+
+
+## Explanation
 
-<!-- Please list affected APIs -->
+<!-- Summary of the request. Please provide a code sample if you have one. -->
```

Comment:
🤔 can you tell me a little bit about what you'd expect to see in the code sample? Not clear how that makes sense to add here.

Reply:
I honestly don't know, now that I've thought about it. I think it's fine without asking for this.